itest/utils.go

```diff
@@ -158,7 +158,7 @@ func MineBlocks(t *testing.T, client *rpcclient.Client,
 		)
 		require.NoError(t, err)
 
-	case rpcclient.BtcdVersion:
+	case *rpcclient.BtcdVersion:
```

Comment:
It seems like this is _NOT_ a pointer, while the `BitcoindVersion` is (see itest that's currently broken). This is very very weird, no idea why. But I tested it locally with the load test and integration test, and it looks like that's really the case...

Reply:
I had to log the `type` to see that it was a pointer in the `bitcoind` case. Just assumed it would also be for the `btcd` case. That is strange.